tests/plugins/test_pull_base_image.py

```diff
@@ -700,20 +700,19 @@ def workflow_callback(workflow):
             test_vals['workflow'] = workflow
             return workflow
 
-        try:
-            test_pull_base_image_plugin(LOCALHOST_REGISTRY, BASE_IMAGE,
-                                        [], [], reactor_config_map=True,
-                                        inspect_only=False,
-                                        workflow_callback=workflow_callback,
-                                        check_platforms=True,  # orchestrator
-                                        )
-        except PluginFailedException as e:
-            if fail and "Missing arches in manifest list" in str(e):
-                # when manifest_list is empty _validate_platforms_in_image raises an assertion
-                # error. We don't care about that in this test
-                pass
-            else:
-                raise
+
+        # platform validation will fail if manifest is missing, skip this
+        # in the test to be able to continue testing parent_images_digests
+        (flexmock(PullBaseImagePlugin)
```

Comment:
Any `flexmock` experts here who could help to make this working? `_validate_platforms_in_image` is still executed in original form and causes test to fail

Reply:
Basically even this example https://flexmock.readthedocs.io/en/latest/user-guide/#class-level-mocks is not working for me, I'm getting G. Bush at the end

Reply:
resolved